packages/frontend/src/components/events/attesterEvents/AttestationCard.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react';
```

Comment:
The directory structure for `src/components` is very deep. Generally we want to avoid that, especially at the start of the project. Better to have one directory with lots of files, then break it into sub-directories as needed.

Reply:
another assumption I got backwards I guess ; )
I will flatten it.